### PR TITLE
Feature/sections/banner

### PR DESCRIPTION
**Pull request by Vomcat**

### CODE REVIEW

**Review by TomaszSack — APPROVED** [review]
all looks good

File: src/components/Banner/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+} from "../../styles";
+
+import FoodImage from "../../assets/images/BannerImage.png";
+import FoodImageMobile from "../../assets/images/BannerImageMobile.png";
+
+const BannerMain = styled.section`
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  padding: 22px 0;
```
Comment (magnowak) [inline src/components/Banner/Banner.tsx]: any chance this could be 24px?

File: src/components/Banner/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+const Banner = () => {
+  return (
+    <BannerMain>
+      <PageContentContainer>
+        <BannerMainWrapper>
+          <BannerHeading>
+            <Subtitle>Welcome to</Subtitle>
+            <Title>
+              <span>Papu</span> Store
```
Comment (magnowak) [inline src/components/Banner/Banner.tsx:95]: is it on purpose that only "Papu" is in <span> and "Store" not?

File: src/components/layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import Instagram from "../../assets/svg/Instagram.svg";
+import Twitter from "../../assets/svg/Twitter.svg";
+
+const FooterMain = styled.footer`
+  width: 100%;
+  background-color: ${colors.white};
+  padding: 20px 0;
+
+  ${respondFrom(breakpoints.desktop)`
+        padding: 33px 0;
```
Comment (magnowak) [inline src/components/layout/Footer.tsx]: those padding values are quite inconsistent, I would stick to multiplies of 8, so 32px

File: src/styles/GlobalStyles.tsx [hunk 4]
```diff
@@ -57,7 +57,8 @@ body{
   font-family: ${fonts.primary};
   line-height: 1;
   color: ${colors.dark};
-  background-color: ${colors.white}
+  background-color: ${colors.white};
+  padding-top: 61px;
```
Comment (magnowak) [inline src/styles/GlobalStyles.tsx]: again ;)

File: src/styles/mixins.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ export const respondFrom = (breakpoints: number) => {
 
 export const respondTo = (breakpoints: number) => {
   return (style: any) =>
-    `@media screen and (max-width: ${breakpoints}px)  { ${style} }`;
+    `@media screen and (max-width: ${breakpoints - 1}px)  { ${style} }`;
```
Comment (magnowak) [inline src/styles/mixins.tsx]: no to mnie zaintrygowało :D

**Review by magnowak — APPROVED** [review]
Looks good minor improvements suggested.

File: src/components/Banner/Banner.tsx (hunk 2 above)
Comment (Vomcat) [reply src/components/Banner/Banner.tsx:95]: Yes

File: src/components/Banner/Banner.tsx (hunk 1 above)
Comment (Vomcat) [reply src/components/Banner/Banner.tsx]: I change it to new spacing variable based on 8 px

File: src/components/layout/Footer.tsx (hunk 3 above)
Comment (Vomcat) [reply src/components/layout/Footer.tsx]: I change it to new spacing variable based on 8 px

**Pull request merged**